### PR TITLE
Add test for inserting the WELPI keyword

### PR DESCRIPTION
**Pull request by lisajulia**

### CODE REVIEW

File: actionx/ACTIONX_WELPI.DATA [hunk 1]
```diff
@@ -305,10 +305,6 @@ SUMMARY
 --
 --       ACTIVATE COLUMNAR SUMMARY DATA REPORTING OPTION
 --
-RUNSUM
---
---       ACTIVATE SUMMARY DATA RSM FILE OUTPUT OPTION
---
```
Comment (blattms) [inline actionx/ACTIONX_WELPI.DATA:311]: Not an expert here. Why is this removed?

Comment (lisajulia) [reply actionx/ACTIONX_WELPI.DATA:311]: [Bard said](https://github.com/OPM/opm-tests/pull/1157#discussion_r1596601980) this summary keyword has created problems sometimes in the past when comparing two runs of flow and he suggested to remove it.

Comment (blattms) [reply actionx/ACTIONX_WELPI.DATA:311]: Yep, just noticed that myself while looking at the history.
